openapi/openapi-proposal.yaml

```diff
@@ -126,20 +130,18 @@ paths:
       - name: depth
         in: query
         description: The number of eras to query for payouts of. Must be less
-          than or equal to `HISTORY_DEPTH`.
+          than or equal to `HISTORY_DEPTH`. Defaults to 1.
```

Comment:
OpenAPI has a field for this. https://swagger.io/docs/specification/describing-parameters/